lib/testing/src/lib/process-services-cloud/resources/resources.ts

```diff
@@ -26,7 +26,8 @@ export const ACTIVITI_CLOUD_APPS: any = {
             anotherCandidateGroupProcess: 'anothercandidategroup',
             uploadFileProcess: 'uploadfileprocess',
             processwithstarteventform: 'processwithstarteventform',
-            processwithjsonfilemapping: 'processwithjsonfilemapping'
+            processwithjsonfilemapping: 'processwithjsonfilemapping',
+            assigneeProcess: 'assigneeprocess'
```

Comment:
Lot of existing processes in simple app are already processes with hruster as assignee. I think, there is no need to add another resource.

Reply:
I remember i couldn't find a very basic process with only hruser as assignee. Can you indicate one please?

Reply:
I see 'calledprocess' which can be used.

Reply:
Oh, i didn't check simpleapp. I will use 'calledprocess' process in this test, thanks, but i will leave 'assigneeProcess' here since the candidateuserapp zip already contains this process.